scripts/build-controller-release.sh

```diff
@@ -86,7 +86,7 @@ Environment variables:
                                         Default: $ACK_GENERATE_SERVICE_ACCOUNT_NAME
   AWS_SDK_GO_VERSION:                   Overrides the version of github.com/aws/aws-sdk-go used
                                         by 'ack-generate' to fetch the service API Specifications.
-                                        Default: $AWS_SDK_GO_VERSION
+                                        Default: Version of aws-controllers-k8s/runtime in go.mod
```

Comment:
is this correct? "aws-controllers-k8s/runtime"? I thought it was service-controller repository

Reply:
Oh this is meant to be `aws/aws-sdk-go` version. Thanks